tests/functional/test_wheel.py

```diff
@@ -215,7 +215,7 @@ def test_pip_wheel_with_pep518_build_reqs(script, data):
 
 @pytest.mark.network
 def test_pip_wheel_with_pep518_build_reqs_no_isolation(script, data):
-    script.pip('install', 'wheel')
+    script.pip('install', '-f', data.find_links, 'wheel', 'simplewheel==2.0')
```

Comment:
This can be split into 2 -- use of common_wheels to install wheel and pip_install_local for installing simplewheel.

Reply:
Will do.